core/src/main/java/com/sequenceiq/cloudbreak/controller/StackController.java

```diff
@@ -210,10 +210,10 @@ private void decorateSubnetEntities(List<Subnet> subnetList, Stack stack) {
 
     private ResponseEntity<IdJson> createStack(CbUser user, StackRequest stackRequest, boolean publicInAccount) {
         Stack stack = conversionService.convert(stackRequest, Stack.class);
+        MDCBuilder.buildMdcContext(stack);
         stack = stackDecorator.decorate(stack, stackRequest.getCredentialId(), stackRequest.getConsulServerCount(), stackRequest.getNetworkId());
```

Comment:
please remove building the MDC context in callers (where the context is built using the user only)